src/containers/HSM/HSMList/HSMList.tsx

```diff
@@ -432,7 +432,7 @@ export const HSMList = () => {
         loadingList={syncTemplateLoad}
         helpData={templateInfo}
         secondaryButton={secondaryButton}
-        title={'Templates'}
+        title={'HSM Template'}
```

Comment:
```suggestion
        title={'HSM Templates'}
```

Please update all instances where we were using plural earlier